bitnami/contour/templates/_helpers.tpl

```diff
@@ -33,6 +33,33 @@ Create the name of the contour-certgen service account to use
 {{- end -}}
 {{- end -}}
 
+{{/*
+Whether to enabled contour-certgen or not
+*/}}
+{{- define "contour.contour-certgen.enabled" -}}
+{{- if and (not .Values.tlsExistingSecret) (or (not .Values.contour.tlsExistingSecret) (not .Values.envoy.tlsExistingSecret))  -}}
+    true
+{{- else -}}{{- end -}}
+{{- end -}}
+
+{{/*
+Contour certs secret name
+*/}}
+{{- define "contour.contour.certs-secret.name" -}}
+{{- $existingSecret := default .Values.tlsExistingSecret .Values.contour.tlsExistingSecret -}}
+{{- $name := default "contourcert" $existingSecret -}}
+{{- printf "%s" $name -}}
+{{- end -}}
+
+{{/*
+Envoy certs secret name
+*/}}
+{{- define "contour.envoy.certs-secret.name" -}}
```

Comment:
Is this being used anywhere? I cannot find any match.

Reply:
`bitnami/contour/templates/envoy/daemonset.yaml`